src/late_multicellular_stage/editor/MetaballEditorComponentBase.cs

```diff
@@ -721,6 +722,34 @@ protected void RunWithSymmetry(Vector3 position, TMetaball? parent, Action<Vecto
                 break;
             }
 
+            case HexEditorSymmetry.XAxisSymmetry:
+            {
+                var symmetryPosition = position * new Vector3(-1, 1, 1);
+
+                if (position.DistanceTo(symmetryPosition) < diameter / 2)
+                {
+                    callback(position, parent);
+                }
+                else
+                {
+                    callback(position, parent);
+
+                    if (parent != null)
+                    {
+                        var symmetryParent = (TMetaball?)editedMetaballs.GetClosestMetaballToPosition(
+                            parent.Position * new Vector3(-1, 1, 1));
+
+                        callback(symmetryPosition, symmetryParent);
+                    }
+                    else
+                    {
+                        callback(symmetryPosition, parent);
+                    }
+                }
+
+                break;
+            }
+
             default:
                 throw new NotSupportedException("symmetry editing not implemented yet");
```

Comment:
Also, I think that this error message should be updated to mean that some symmetry types aren't supported, as opposed to symmetry not being implemented